docs/upgrading.asciidoc

```diff
@@ -37,3 +42,90 @@ The table below is a simplified description of this policy.
 |2.0.x |2020-05-14 |2.1.0
 |1.0.x |2020-02-23 |3.0.0
 |====
+
+[[upgrade-to-v5]]
+=== Upgrade to version 5.x
+
+Upgrading from version `4.x` to `5.x` of the RUM Agent introduces some breaking changes.
+This guide outlines the changes and the steps required to upgrade.
+
+[float]
+[[v5-breaking-changes]]
+=== 5.0 Breaking changes
+
+*Agent name changed to `rum-js`*
+
+The Agent name has been changed to `rum-js`.
+Because of this change, you may need to upgrade your APM Server instance,
+and therefor your Elasticsearch and Kibana instance.
```

Comment:
Maybe mention the reason users need to upgrade their stack specially kibana. The reason is that kibana UI recognises the RUM agent name and shows RUM specific visualisation. prior to 7.x release, kibana only recognised the old name (i.e `js-base`).

Reply:
Perfect, thanks